README.rst

```diff
@@ -108,27 +108,21 @@ first name begins with 'D':
 
 .. code-block:: python
 
-    for user in UserModel.query("John", first_name__begins_with="D"):
+    for user in UserModel.query("John", filter_condition=(UserModel.first_name.startswith("D")):
         print(user.first_name)
 
 Examples of ways to query your table with filter conditions:
 
 .. code-block:: python
 
-    for user in UserModel.query("John", filter_condition= (UserModel.email=="djohn@company.org")):
+    for user in UserModel.query("John", filter_condition=(UserModel.email=="djohn@company.org")):
```

Comment:
same as above: `UserModel.query("John", UserModel.email=="djohn@company.org")`
i know it's just my opinion but i think it reads better